packages/yoastseo/src/languageProcessing/helpers/sentence/SentenceTokenizer.js

```diff
@@ -327,6 +334,26 @@ export default class SentenceTokenizer {
 		}
 	}
 
+	/**
+	 * Checks if a string ends with an abbreviation.
+	 * @param {string} currentSentence A (part of) a sentence.
+	 * @returns {boolean} True if the string ends with an abbreviation that is in abbreviations.js. Otherwise False.
+	 */
+	endsWithAbbreviation( currentSentence ) {
+		const abbreviationsMatch = currentSentence.match( abbreviationsRegex );
+
+		if ( ! abbreviationsMatch ) {
+			return false;
+		}
+
+		// Only return true if it the last word is an abbreviation.
+		const abbreviationsMatchString = abbreviationsMatch.slice( -1 )[ 0 ].trim();
+		const matchLength = abbreviationsMatchString.length;
+		const tokenStringEnd = currentSentence.slice( currentSentence.length - matchLength, currentSentence.length ).trim();
```

Comment:
This logic is a bit messy, I used this to be able to use the `createRegexFromArray` function.

Reply:
Yeah, I think this can be simplified! I've just committed an attempt to do so.